### PR TITLE
Apollo!

### PR DESCRIPTION
**Pull request by arfon**
Replaces #3112

### CODE REVIEW

File: lib/linguist/languages.yml [hunk 1]
```diff
@@ -188,6 +188,15 @@ Apex:
   tm_scope: source.java
   ace_mode: java
 
+Apollo Guidance Computer:
+  type: programming
+  color: "#0B3D91"
+  group: Assembly
+  extensions:
+  - .agc
+  tm_scope: source.assembly
```
Comment (arfon) [inline lib/linguist/languages.yml]: @Alhadis - should this be `source.agc`

Comment (Alhadis) [inline lib/linguist/languages.yml]: CRAP. Forgot about that! Yes, definitely.

**Comment by arfon** [comment]
Not sure what is up with Travis today but the builds are taking _forever_. Tests are passing locally for me so merging this.

**Comment by Alhadis** [comment]
> Not sure what is up with Travis

[Still in hospital](https://github.com/github/linguist/pull/3105#issuecomment-231702259).

**Pull request merged**